src/org/intellij/erlang/stubs/ErlangSpecificationStub.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2012-2015 Sergey Ignatov
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.intellij.erlang.stubs;
+
+import com.intellij.psi.stubs.IStubElementType;
+import com.intellij.psi.stubs.StubElement;
+import com.intellij.util.io.StringRef;
+import org.intellij.erlang.psi.ErlangSpecification;
+
+public class ErlangSpecificationStub extends ErlangStringContainerStub<ErlangSpecification> {
```

Comment:
I was thinking about ErlangWithArityStub, but it requires element instanceof ErlangNamedElement. ErlangSpecification is not ErlangNamedElement.
I can fix it, but that way I must implement methods setName, getNameIdentifier. I think it's too much. 
That's why I prefer ErlangStringContainerStub. 
Yes, now I see it was bad solution.

I should probably extend StubBase. 
What do you think is better - StubBase or ErlangWithArityStub?